.travis.yml

```diff
@@ -81,15 +87,15 @@ before_script:
    # Now build pFUnit
    - cd ${TRAVIS_BUILD_DIR}
    - mkdir -p build && cd build
-   - cmake .. -DCMAKE_Fortran_COMPILER=${FC} -DCMAKE_INSTALL_PREFIX=${HOME}/Software/pFUnit
+   - cmake .. -DCMAKE_Fortran_COMPILER=${FC} -DCMAKE_BUILD_TYPE=Debug -DCMAKE_INSTALL_PREFIX=${HOME}/Software/pFUnit
 
 script:
    # Build
    - make -j$(nproc) VERBOSE=1
    # Make tests
    - make -j$(nproc) tests
-   # Run tests. If the first fails, run with -VV to try and diagnose errors
-   - ctest -j $(nproc) || ctest -VV
+   # Run tests
+   - ctest -j $(nproc) --output-on-failure
```

Comment:
Nice.  Did not know about that option.   @mathomp4 please take note for our other projects

Reply:
Neither did I. I like it.